core/src/main/java/org/infinispan/interceptors/impl/ClusteringInterceptor.java

```diff
@@ -81,7 +81,7 @@ protected Boolean addException(Address sender, Exception exception) {
 
       @Override
       protected final Boolean addValidResponse(Address sender, ValidResponse response) {
-         return (Boolean) response.getResponseValue();
+         return addBooleanResponse(sender, (Boolean) response.getResponseValue());
```

Comment:
This is the main crux of the failures as scattered would always return FALSE if the non backup replied with FALSE first. By fixing this is shows the other failures in the collector and test.